fiberzap/README.md

```
@@ -95,7 +95,7 @@
 
 import (
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)